### PR TITLE
Setup for signatures sync using AWS Glue

### PR DESCRIPTION
**Pull request by anero**
Adds example to use new AWS Glue resources from `controlshift/controlshift-redshift-sync/aws` module.

### CODE REVIEW

File: warehouse.tf [hunk 1]
```diff
@@ -44,17 +45,22 @@ data "aws_iam_policy_document" "redshift_load_policy" {
...
+  name = "subnet-group-for-redshift-cluster"
+  subnet_ids = [ aws_subnet.private_redshift_subnet_shared_with_lambdas_and_glue.id ]
+}
+
+resource "aws_security_group" "allow_access_to_redshift_from_glue" {
+  name        = "Allow AWS Glue access to Redshift"
+  description = "Allow inbound access from all servers within the security group, and allow full outbound access"
   vpc_id      = data.aws_vpc.default.id
 
   ingress {
```
Comment (woodhull) [inline warehouse.tf:58]: What is this used for exactly, allowing external access to our test example redshift? Can we replace with a variable to force people using this example plan to make a choice about who has access?

Comment (woodhull) [reply warehouse.tf:58]: compared to the description this seems to allow access from the entire internet, not just glue subnet.

**Review by woodhull — APPROVED** [review]
Just one note re: permissions.

File: warehouse.tf (hunk 1 above)
Comment (anero) [reply warehouse.tf:58]: The `self = true` on the route means it only allows traffic from the same security group, so this isn't limiting by IP address but by internal traffic (see docs for `ingress` block at https://www.terraform.io/docs/providers/aws/r/security_group.html)

Comment (anero) [reply warehouse.tf:58]: And here's Glue docs with the requirement to allow full access within the security group: https://docs.aws.amazon.com/glue/latest/dg/setup-vpc-for-glue-access.html

**Pull request merged**